torch/onnx/symbolic_opset10.py

```diff
@@ -324,7 +324,7 @@ def embedding_bag(
     include_last_offset,
     padding_idx,
 ):
-    if scale_grad_by_freq and GLOBALS.training_mode:
+    if scale_grad_by_freq and GLOBALS.training_mode == _C_onnx.TrainingMode.TRAINING:
```

Comment:
will you revert `GLOBALS.training_mode` to boolean or keep it as enum? Current enum setting seems to have an issue with the case when `mode == PRESERVE and original_export is True`.

Reply:
Currently I am keeping it as enum. `mode == PRESERVE and originally_training is True` is the case I was concerned about. WIth this we treat them as EVAL mode even when originally_training is True. What do you think?

Reply:
Updated